src/main/java/com/openshift/jenkins/plugins/pipeline/MessageConstants.java

```diff
@@ -0,0 +1,60 @@
+package com.openshift.jenkins.plugins.pipeline;
+
+public class MessageConstants {
+
+public static final String CANNOT_GET_CLIENT = "\n\nExiting \"%s\" unsuccessfully; a client connection to \"%s\" could not be obtained.";
+public static final String BAD_BUILD_STATE = "\n\nExiting \"%s\" unsuccessfully; build \"%s\" has completed with status:  [%s].";
+public static final String GOOD_BUILD_WITH_DEPLOY = "\n\nExiting \"%s\" successfully; build \"%s\" has completed with status:  [Complete].  All deployments with ImageChange triggers based on this build's output triggered off of the new image.";
+public static final String GOOD_BUILD_BAD_DEPLOY = "\n\nExiting \"%s\" unsuccessfully; build \"%s\" has completed with status:  [Complete]. However, not all deployments with ImageChange triggers based on this build's output triggered off of the new image.";
+public static final String GOOD_BUILD_IGNORE_DEPLOY = "\n\nExiting \"%s\" successfully; build \"%s\" has completed with status:  [Complete].";
+public static final String START_BUILD_CANCEL = "\n\nStarting the \"%s\" action for build config \"%s\" from the project \"%s\".";
```

Comment:
Been ruminating a bit on this since I saw your response last night.  Where I ended up was that my use of %s went off in the weeds a bit as I worried about consolidating strings and substituting in things like the step/action name (where that got tedious).

Where my mindset is now:
- have explicit strings for each step/action (but I'll still leverage constants for the step name)
- don't use %s for plugging in step/action name or for conditional checks (i.e. the build 
- only use %s for dynamic from the api server ... the state of something (failed or complete), or the name of a build or deployment

Couldn't tell for sure if case action vs. step was still a point of confusion.  Just in case:
- "step" is used for the selectable items from the "build step" drop down on the Jenkins UI
- "action" is used for the selectable items from the "post-build action" drop down